### ISSUE TITLE
Any consideration to incorporate the list into sindresorhus/awesome?

### ISSUE THREAD

**Issue by batrdn** [oz123/awesome-c#316]
I noticed that the [awesome-c](https://github.com/inputsh/awesome-c#readme) list in the [sindresorhus/awesome](https://github.com/sindresorhus/awesome) is no longer maintained and already archived. Since this is the most up-to-date list and has relatively a lot more stars than the current one, are you considering to include it into that repo?

**Comment by oz123** [comment]
I wasn't aware of it, and since it's not my repo there I can only make a PR.

**Comment by batrdn** [comment]
@oz123 yes exactly, but you might also wanna follow their guidelines, and one that already conflicts within this repository is, for example, the name of the main branch, which is `master` and they want it set as `main`.

**Comment by oz123** [comment]
Well, their guidelines are quite demanding, which I understand. Will have to wait for a time when I am more available to go through all of it.
